### PR TITLE
[release] 1.5.0

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Review by ronronscelestes — APPROVED** [review]

**Review by gu-stav — APPROVED** [review]

File: packages/strapi-design-system/src/DatePicker/DatePicker.js [hunk 1]
```diff
@@ -115,6 +113,7 @@ DatePicker.defaultProps = {
   minDate: undefined,
   maxDate: undefined,
   onClear: undefined,
+  parentFieldId: undefined,
```
Comment (gu-stav) [inline packages/strapi-design-system/src/DatePicker/DatePicker.js:116]: This shouldn't be here.

File: packages/strapi-design-system/src/DatePicker/DatePicker.js [hunk 2]
```diff
@@ -132,6 +131,7 @@ DatePicker.propTypes = {
   minDate: PropTypes.instanceOf(Date),
   onChange: PropTypes.func.isRequired,
   onClear: PropTypes.func,
+  parentFieldId: PropTypes.string,
```
Comment (gu-stav) [inline packages/strapi-design-system/src/DatePicker/DatePicker.js:134]: Neither this.

**Review by gu-stav — CHANGES_REQUESTED** [review]

File: packages/strapi-design-system/src/DatePicker/DatePicker.js (hunk 1 above)
Comment (gu-stav) [reply packages/strapi-design-system/src/DatePicker/DatePicker.js:116]: @ronronscelestes Could you take care of this, please? 🙏🏼

File: packages/strapi-design-system/src/DatePicker/DatePicker.js (hunk 2 above)
Comment (gu-stav) [reply packages/strapi-design-system/src/DatePicker/DatePicker.js:134]: @ronronscelestes Could you take care of this, please? 🙏🏼

Comment (ronronscelestes) [reply packages/strapi-design-system/src/DatePicker/DatePicker.js:134]: on it, my bad

**Review by ronronscelestes — APPROVED** [review]

**Pull request merged**